### ISSUE TITLE
Ability send users hugs 🤗

### ISSUE THREAD

**Issue by joeworkman** [yourhead/s3#781]
Some of them just need that from time to time...

**Comment by joeworkman** [comment]
I like that this is still open. Must mean that you are thinking about it...

**Comment by yourhead** [comment]
You’re right. They do need hugs sometimes. Sending hugs to the user seems like a great idea.

**Comment by joeworkman** [comment]
Yes. Hugs... 🤗
